libs/langchain/langchain/embeddings/__init__.py

```diff
@@ -23,6 +23,7 @@
 from langchain.embeddings.baidu_qianfan_endpoint import QianfanEmbeddingsEndpoint
 from langchain.embeddings.bedrock import BedrockEmbeddings
 from langchain.embeddings.bookend import BookendEmbeddings
+from langchain.embeddings.bytedance_volcano import VolcanoEmbeddings
```

Comment:
no need to do anything in `langchain`. existing imports are kept for backwards compatibility.

Reply:
fixed